packages/snaps-execution-environments/coverage.json

```diff
@@ -1,6 +1,6 @@
 {
-  "branches": 81.29,
-  "functions": 92.59,
-  "lines": 92.03,
-  "statements": 91.71
+  "branches": 80.85,
```

Comment:
Any clue why some of these coverage numbers have droped?

Reply:
I think because the function signature changed for the endowment function. I can't spot any untested new code.

Reply:
Yea, that seems reasonable until we have 100% of coverage in this repo 😅